### PR TITLE
[IMP] sales: updated get signature for validation doc for RST formatting

### PR DESCRIPTION
**Pull request by ksc-odoo**
updated RST formatting and compressed images (added additional image, as well) to the "get signature for validation" sales doc for v14 - but should be forward-ported to 15, 16 (and minor modifications and adjustments will be made to those versions in the future)

Project Task: https://www.odoo.com/web#id=3330435&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form

### CODE REVIEW

File: content/applications/sales/sales/send_quotations/get_signature_to_validate.rst
Comment (hojo-odoo) [inline content/applications/sales/sales/send_quotations/get_signature_to_validate.rst:24]: This conditional doesn't make sense. It reads like you're saying that the Online Signature feature is found on the Confirmation tab IF the reader is familiar with the quote template documentation. I think you can remove the intro entirely. See below.
```suggestion
   The :guilabel:`Online Signature` feature for each quotation template is found under the
   :guilabel:`Confirmation` tab.
```

File: content/applications/sales/sales/send_quotations/get_signature_to_validate.rst
Comment (hojo-odoo) [inline content/applications/sales/sales/send_quotations/get_signature_to_validate.rst:41]:
```suggestion
on the quotation. When clicked, clients have the option to draw a signature, fill in
```

File: content/applications/sales/sales/send_quotations/get_signature_to_validate.rst
Comment (samueljlieber) [inline content/applications/sales/sales/send_quotations/get_signature_to_validate.rst:16]: Early line break
```suggestion
Settings`, scroll to the :guilabel:`Quotations & Orders` heading, and activate the :guilabel:`Online
Signature` feature by checking the box beside it, and clicking the :guilabel:`Save` button in the
top-left corner.
```

File: content/applications/sales/sales/send_quotations/get_signature_to_validate.rst
Comment (samueljlieber) [inline content/applications/sales/sales/send_quotations/get_signature_to_validate.rst:13]: No need to capitalize *online signature*
```suggestion
To activate the *online signature* feature, go to :menuselection:`Sales app --> Configuration -->
```

File: content/applications/sales/sales/send_quotations/get_signature_to_validate.rst
Comment (samueljlieber) [inline content/applications/sales/sales/send_quotations/get_signature_to_validate.rst:20]: All three following images have the same alt text, its best for each to be descriptive and unique.
```suggestion
   :alt: How to enable online signature in Odoo Sales settings.
```

File: content/applications/sales/sales/send_quotations/get_signature_to_validate.rst
Comment (samueljlieber) [inline content/applications/sales/sales/send_quotations/get_signature_to_validate.rst:28]:
```suggestion
      :alt: How to enable online signature on a quotation template.
```

File: content/applications/sales/sales/send_quotations/get_signature_to_validate.rst
Comment (samueljlieber) [inline content/applications/sales/sales/send_quotations/get_signature_to_validate.rst:35]:
```suggestion
      :alt: How to enable online signature on standard quotations.
```

**Review by samueljlieber — CHANGES_REQUESTED** [review]
Hi @ksc-odoo, nice job! I have just a couple of changes needed, let me know when this PR is ready for another look!

**Comment by ksc-odoo** [comment]
> Hi @ksc-odoo, nice job! I have just a couple of changes needed, let me know when this PR is ready for another look!

Alrighty, @samueljlieber - this is ready for another quick review, whenever you get a chance. Thanks! 👍

**Review by samueljlieber — APPROVED** [review]
Hi @ksc-odoo this PR looks great! Approving now and sending to final review. @StraubCreative this should be a quick merge 🙂

File: content/applications/sales/sales/send_quotations/get_signature_to_validate.rst
Comment (meng-odoo) [inline content/applications/sales/sales/send_quotations/get_signature_to_validate.rst:23]: Changing the sentence structure slightly to mirror the following sentence and draw the comparison more clearly.
```suggestion
   On quotation templates, the :guilabel:`Online Signature` feature is located under the
```

File: content/applications/sales/sales/send_quotations/get_signature_to_validate.rst
Comment (meng-odoo) [inline content/applications/sales/sales/send_quotations/get_signature_to_validate.rst:49]:
```suggestion
and confirmed, a delivery order is automatically created (if the Odoo *Inventory* app is installed).
```

File: content/applications/sales/sales/send_quotations/get_signature_to_validate.rst
Comment (meng-odoo) [inline content/applications/sales/sales/send_quotations/get_signature_to_validate.rst:41]:
```suggestion
on the quotation in the online customer portal. When clicked, clients have the option to draw a signature, fill in the field with
```

**Review by meng-odoo — CHANGES_REQUESTED** [review]
Hi @ksc-odoo, this doc looks great! I just had a few tiny changes before we merge. Thanks!

**Comment by ksc-odoo** [comment]
> Hi @ksc-odoo, this doc looks great! I just had a few tiny changes before we merge. Thanks!

@meng-odoo Alrighty - just made all those suggested edits. Thanks for the super-helpful feedback! If there's anything else you need me to do with this doc - just let me know. If not, feel free to merge! Thanks again 👍

**Comment by meng-odoo** [comment]
@samueljlieber There were only very tiny changes to this PR, but could you please do another quick review before we merge? Thanks so much :)

**Review by samueljlieber — APPROVED** [review]
Hi @ksc-odoo this PR looks good to me! Nice job 🙂 

@StraubCreative ready for merge 👍 and can be FWP to 15.0 & 16.0.

**Review by StraubCreative — APPROVED** [review]
Short and sweet, nice job @ksc-odoo 👍 
Approving with two small comments which can optionally be addressed later

@robodoo r+

**Pull request closed without merging**
